CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- the value set as `predefinedItemsPerPage` to the pagination dropdown
```

Comment:
can you modify this to reflect the actual change? :-) 

Something like `update react-pattern-ui to the latest version (2.2.1)`

Reply:
@neotrow Done :D